File: fullFrame.cpp
#include<iostream>
#include<cmath>
using namespace std;
int main(){
	int number;
	cout << "Enter a number = ";
	cin >> number;
	int a, b;
	a = sqrt(number);
	b = a * a;
	cout << ((number == b) ? "Full frame" : "Not full frame");
	return 0;
}

File: fullFrame.java
package springIntro;

import java.util.*;

public class fullFrame {

	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		System.out.println("Enter a number = ");
		int number = in.nextInt();
		int sqrtN;
		sqrtN = (int) Math.sqrt(number) * (int) Math.sqrt(number);
		System.out.println(number == sqrtN ? "Full frame" : "Not full frame");
	}

}
